docs/Installation.rst

```diff
@@ -263,23 +389,54 @@ its own library file. An example of it is given in *IntegrationTestFiles/testLib
   1001.03c;   4069;    <absolute_path>/1001JEF33.ace;
   ...
 
-`Line Number` is the line in the file at which a particular data card begins. Each line cannot
-contain more then a single entry. Each component must be delimited by a semi-colon.
+Here, ``Line Number`` is the line in the file at which a particular data card begins. Each line cannot
+contain more than one entry, and each component must be delimited by a ';'. An example of such a file 
+is given in *IntegrationTestFiles/testLib*.
 
-To generate the library file from the collection of raw ACE files one can use the
-``scripts/make_ace_lib.sh`` bash script. It can be run with the following command:
+To generate the library file from the collection of downloaded raw ACE files, one can use the
+``scripts/make_ace_lib.sh`` bash script, which can be run using the following command (to get extra 
+help, run the script without any arguments):
 
 .. code-block:: bash
 
   ./scripts/make_ace_lib.sh /path/lib.xsfile CE ./path_to_ace_files/*.ace
 
-To get extra help run the script without any arguments. The ``CE`` letters allow to select between
-searching for continuous energy neutron data cards and thermal scattering S(α,β) cards (SAB mode).
-Sadly the script can search only for a single type of card in one pass. Thus to create a full
-library with thermal data we need to do the following:
+The ``CE`` letters allow to switch between searching for continuous energy (CE) and thermal scattering S(α,β) 
+(SAB) neutron data cards. Sadly, the script can search only for a single card type in one pass; thus, to create 
+a full library with thermal scattering data included one needs to run the following:
 
 .. code-block:: bash
 
   ./scripts/make_ace_lib.sh ./tempCE CE ./path_to_CE_ace_files/*.ace
   ./scripts/make_ace_lib.sh ./tempSAB SAB ./path_to_SAB_ace_files/*.ace
   cat tempCE tempSAB > fullLib.xsfile
+
+Running your first simulation with SCONE
+----------------------------------------
+
+Once the ``fullLib.xsfile`` has been generated, we can run our first actual simulation. SCONE uses text files as
+simulation inputs. Instances of such files are included in the ``InputFiles`` directory. For this example, we will
+use the `JEZ` input file. Navigate to the ``InputFiles`` directory, open the `JEZ` file and locate the following
+lines::
+     
+     nuclearData {
+
+          handles {
+               ceData { type aceNeutronDatabase; aceLibrary $SCONE_ACE;}
+          }
+
+          ...
+
+You may replace the ``$SCONE_ACE`` environmental variable in the `JEZ` file with the absolute path to the ``fullLib.xsfile`` 
+file or, better yet, export this variable in your ``.bashrc``/``.zprofile`` file (depending on your OS) by adding the 
+following line::
+     
+     export SCONE_ACE=path_to_fullLib.xsfile
+
+and saving the changes in either case. Note that if you export this variable in your ``.bashrc``/``.zprofile`` file, you
```

Comment:
I wouldn't insist on setting the environmental variables in the `.basrc`. I think it is fine to export them before running. It makes (should make) one more conscious of the nuclear data they use ;-)